AUTHORS.md

```diff
@@ -10,4 +10,4 @@ Contributors
 ------------
```

Comment:
I would add a new section called as Core Devs to include our names, and also keep a `Contributors` one for external contributor (like this case https://github.com/Clariteia/minos-pypackage/blob/main/%7B%7Bcookiecutter.project_slug%7D%7D/AUTHORS.md)

Also, you you forgot to include your own name!!